### ISSUE TITLE
Supports multiple "profiles"

### ISSUE THREAD

**Issue by nt1m** [captainbrosset/domprofiler#14]
It'd be nice to have support multiple "profiles", probably something similar to built-in profiler.

**Comment by captainbrosset** [comment]
Yes, pretty nice indeed. I like this better than #13. Not to say that we shouldn't do #13, but I think we should do this one first.
